### ISSUE TITLE
VuePasswordAuto is not passing NamedSlots to VuePassword

### ISSUE THREAD

**Issue by marcelo-rebello** [skegel13/vue-password#23]

**Comment by marcelo-rebello** [comment]
VuePasswordAuto is not passing named slots to VuePassword, so it is impossible to change the UI of the input component.

some code like that will be usefull in template:

```
   <template v-for="(_, name) in $scopedSlots" v-slot:[name]="slotData">
         <slot :name="name" :props="slotData" v-bind="slotData" />
    </template>
```

**Comment by skegel13** [comment]
Good catch, I have this fixed and in version 2.0.3 (92f6c566787cd1f361d999bde12d6ecc37df8d3b).
